assets/aml-benchmark/tests/pipelines/prompt_crafter_pipeline.yaml

```diff
@@ -18,6 +18,7 @@ inputs:
   output_pattern: " {{answerKey}} "
   prompt_type: "completions"
   system_message: "Answer truthfully. "
+  ground_truth_column_name: "answerKey"
```

Comment:
Do we have a test that asserts that the ground truth column is in the output if it is passed in?

Reply:
Done. Added multiple tests for it.